Source/TrackableEntities.EF.5/DbContextExtensions.cs

```diff
@@ -207,8 +208,16 @@ public static void ApplyChanges(this InterceptorPool pool, IEnumerable<ITrackabl
                 {
                     // Mark modified properties
                     SetEntityState(context, item, parent, propertyName, EntityState.Unchanged, interceptors);
-                    foreach (var property in item.ModifiedProperties)
-                        context.Entry(item).Property(property).IsModified = true;
+                    var entry = context.Entry(item);
+                    // Exclude properties attributed as NotMapped
+                    var notMappedProps = item.GetType()
```

Comment:
Yes, if this is handled on the server-side, it will have to be EDM-based, because non-mapped properties can be set via fluent API.

If this is too difficult, I would suggest instead doing this on the client-side, with an extension method that looks for the [NotMapped] attribute using reflection and then adds those properties to `ExcludedProperties` on the CTC.